depends/packages/packages.mk

```
@@ -1,6 +1,10 @@
 zcash_packages := libsodium utfcpp
 packages := boost libevent zeromq $(zcash_packages) googletest
-native_packages := native_clang native_ccache native_rust
+native_packages := native_clang native_ccache native_rust 
```

The change to this line should have been undone.
```suggestion
native_packages := native_clang native_ccache native_rust
```

---

thanks for spotting that; i can't reopen this PR (tried both with the web interface and the `gh` command line interface) so i'm going to submit a new PR fixing this

---

https://github.com/zcash/zcash/pull/5600